HeavyFlavorAnalysis/SpecificDecay/plugins/BPHWriteSpecificDecay.h

```diff
@@ -172,6 +172,7 @@ class BPHWriteSpecificDecay:
         const RefCountedKinematicParticle kinPart = ptr->currentParticle();
         const           KinematicState    kinStat = kinPart->currentState();
         cc.addUserFloat( "fitMass", kinStat.mass() );
+        if ( writeMomentum )
```

Comment:
hi @ronchese - given the new if statement, line 176 should be indented further. Please update that line. Thanks